test/Microsoft.AspNetCore.Mvc.FunctionalTests/FiltersTest.cs

```diff
@@ -554,5 +554,53 @@ public FiltersTest(MvcTestFixture<FiltersWebSite.Startup> fixture)
                 $"CurrentCulture:{expected},CurrentUICulture:{expected}",
                 await response.Content.ReadAsStringAsync());
         }
+
+        [Fact]
+        public async Task RequestSizeLimit_ThrowsForInvalidSize()
```

Comment:
Please make a new class for these tests and don't put them in `FiltersWebSite`. That thing is a mess and we plan to just nuke it from orbit some day.